src/nvim/channel.c

```diff
@@ -762,6 +762,14 @@ static void set_info_event(void **argv)
   channel_decref(chan);
 }
 
+bool channel_job_running(uint64_t id)
```

Comment:
Would be nice if we had this earlier. I didn't port job/terminal code from vim patches because I needed functions like this to check if a job/terminal is running.

Reply:
Ok, anything else missing?